api/src/main/java/marquez/api/OpsResource.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018-2024 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.api;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.ResponseMetered;
+import com.codahale.metrics.annotation.Timed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import marquez.service.OpsService;
+import marquez.service.ServiceFactory;
+
+@Slf4j
+@Path("/api/v1/ops")
+public class OpsResource {
```

Comment:
Minor suggestions on naming:

Use endpoint `GET /api/v1/stats/<stats-type>?period=<stats-period>`; where:
* `<stats-type>` - specifies the type of stats to return.
* `<stats-period>` - specifies whether to return statistics for a particular day or week.


this would allow us to provide stats for other in the future by replacing `<stats-type>` with `lineage_events`, `run_history`, `datasets`, etc.

For lineage events stats the would power our events dashboard in our overview page, I would use the API call:

```
GET /api/v1/stats/lineage_events?period=day
```